src/annotator/anchoring/pdf.js

```diff
@@ -442,19 +454,18 @@ async function anchorQuote(quoteSelector, positionHint) {
     }
 
     if (!bestMatch || match.score > bestMatch.match.score) {
+      const [start, end] = translateOffsets(
+        strippedText,
+        text,
+        match.start,
+        match.end,
+        isNotSpace
+      );
```

Comment:
This is performing the opposite translation to the above `translateOffsets` usage: mapping the offset of a quote match from the whitespace-stripped version of a page's text to the whitespace-inclusive version of the page's text.